src/NUnitEngine/mock-assembly-x86/mock-assembly-x86.csproj

```
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RootNamespace>NUnit.Tests</RootNamespace>
+    <TargetFrameworks>net35;net40;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+    <PlatformTarget>x86</PlatformTarget>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NUnit" Version="3.13.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\EngineVersion.cs" LinkBase="Properties" />
+    <Compile Include="..\mock-assembly\AccessesCurrentTestContextDuringDiscovery.cs" Link="AccessesCurrentTestContextDuringDiscovery.cs" />
+    <Compile Include="..\mock-assembly\MockAssembly.cs" Link="MockAssembly.cs" />
```

```suggestion
    <Compile Include="..\mock-assembly\*.cs" />
```

Maintenance suggestion: I believe the above will mean we don't need to add any potential new files to this list in future. 🙂

---

That's OK if we end up seeing the files in SolutionExplorer! I'll give it a try. That said, I think the first file, which is an actual test, is a misuse of mock-assembly, which was only ever intended as fodder for the engine to use in testing loading, running and XML generation... i.e. it's data!